contentreview-impl/pack/src/webapp/WEB-INF/components.xml

```diff
@@ -63,14 +65,15 @@
     </bean>
 
 	<!--  Uncomment this to use a site property to define which sites use c-r -->
-	<!--
-		<bean id="org.sakaiproject.contentreview.service.ContentReviewSiteAdvisor" class="org.sakaiproject.contentreview.impl.adivisors.SitePropertyAdvisor">
+	

```

Comment:
I have left just the site property one. I'm not sure how you're using the other one so if you need them both we should find a way.